tests/js/common/metaschema-tests.js

```diff
@@ -0,0 +1,100 @@
+/*
+
+    Independently of the validator, confirm that our GSS metaschema is itself valid, and that it can be used with Ajv
+    to validate GSS schemas.
+
+*/
+/* globals Ajv, jqUnit, require */
+/* eslint-env browser */
+var fluid_3_0_0 = fluid_3_0_0 || {};
```

Comment:
I think there isn't a need to use the versioned namespace here since this is only of interest to libraries who wish to be versioned along with infusion (which in any case is a scheme we have not bothered to update for many years). The use of plain "fluid" will be fine.

Reply:
I standardised and cleaned this up throughout.